components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -1,108 +1,85 @@
+  [attr.error-message]="getErrorMessage()"
+  (secondaryConfirm)="openModal()"
+  (confirm)="addMembers()"
+  (close)="closePage()">
+    <div *ngIf="!loading">
+      <chef-modal label="member-expression-label" class="member-expression-modal" [visible]="modalVisible" (close)="closeModal()">
+        <h2>Add Member Expression</h2>
+        <div id="left-align">
+          <form [formGroup]="expressionForm">
+            <chef-form-field>
+              <label>Member expression *</label>
+              <input chefInput formControlName="expression" type="text">
+              <chef-error *ngIf="unparsableMember">Invalid member expression.</chef-error>
+              <chef-error *ngIf="duplicateMember">Member expression already in table.</chef-error>
+              <chef-error *ngIf="alreadyPolicyMember">Member already in policy.</chef-error>
+            </chef-form-field>
+          </form>
+          <p><span class="expression">user | team : local | ldap | saml : &lt;name&gt;</span> or <span class="expression">token : &lt;id&gt;</span>, wildcards (*) are allowed.</p>
+          <p>More details in <a href="https://automate.chef.io/docs/iam-v2-guide/#member-expressions" target="_blank">member expression documentation</a>.</p>
+        </div>
+        <div id="align-buttons">
+          <chef-button primary [disabled]="!expressionForm.valid" (click)="validateAndAddExpression()">Add Expression</chef-button>
+          <chef-button tertiary (click)="closeModal()">Cancel</chef-button>
+        </div>
+      </chef-modal>
+      <div id="no-members-container" *ngIf="sortedMembersAvailable.length === 0">
+        <h3>Create more local users or local teams first, or add a member expression.</h3>
+        <chef-button
+          primary
+          class="add-member-button"
+          (click)="openModal()">
+            Add Member Expression
         </chef-button>
       </div>
-    </div>
-    <div id="no-members-container" *ngIf="sortedMembersAvailable.length === 0">
-      <h3>
-        Create more local users or local teams first, or add a member expression.
-      </h3>
-      <chef-button primary class="add-member-button" (click)="openModal()">Add Member Expression</chef-button>
-    </div>
-    <div id="table-container" *ngIf="sortedMembersAvailable.length > 0">
-      <div class="flex">
-        <div id="members-selected" class="flex-right" >
-          {{ membersToAddValues().length | pluralize : 'member' : 's' }} selected
+      <div id="table-container" *ngIf="sortedMembersAvailable.length > 0">
+        <div class="flex">
+          <div id="members-selected" class="flex-right" >
+            {{ membersToAddValues().length | pluralize : 'member' : 's' }} selected
+          </div>
+        </div>
+        <div id="inner-container">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th class="checkbox-row"></chef-th>
+                <chef-th>
+                  ID
+                </chef-th>
```

Comment:
Suggest put element on one line, i.e.: `<chef-th>ID</chef-th>`
Similarly for L57-59.